common/source/docs/common-mount-targeting.rst

```diff
@@ -1,72 +1,84 @@
 .. _common-mount-targeting:
 
-
 =======================
 Gimbal / Mount Controls
 =======================
+ArduPilot provides mechanisms to control the pointing direction (aka targeting) automatically for camera mounts (know hereafter as ``mounts``), as well as by pilot commands/controls. Mounts can be commanded point in at least six different ways (``targeting modes``) by ArduPilot.  This page provides an overview of these controls and their setup.
```

Comment:
small typo, "Mounts can be commanded **to** point in ...".  If you're editing that line, I'd probably also remove "by ArduPilot" because this is an AP wiki so it goes without saying